tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -14719,16 +14720,16 @@ TEST_F(EnergyPlusFixture, Test_UnitarySystemModel_SubcoolReheatCoil)
     EXPECT_EQ(state->dataCoilCooingDX->coilCoolingDXs[0].performance.ModeRatio, 1.0);
     EXPECT_NEAR(thisSys->CoilSHR, thisSys->LoadSHR, 0.001);
     EXPECT_NEAR(SenOutput, -227.705, 0.1);
-    EXPECT_NEAR(LatOutput, -737.9931, 0.1);
+    EXPECT_NEAR(LatOutput, -1531, 0.1);
```

Comment:
Interesting. This greatly changes the coil SHR but shouldn't the load SHR be fixed based on the load? Here it changed with the change in coil performance. Maybe the change in StdBaro did that? But, the coil SHR using these results = 0.129 yet the CoilSHR and LoadSHR = 0.0113 ?

    EXPECT_NEAR(thisSys->CoilSHR, thisSys->LoadSHR, 0.001);
    EXPECT_NEAR(thisSys->CoilSHR, 0.01133, 0.00001);
    EXPECT_NEAR(SenOutput, -227.705, 0.1);
    EXPECT_NEAR(LatOutput, -1531, 0.1);
    EXPECT_NEAR(SenOutput / (SenOutput + LatOutput), 0.129, 0.001);

![image](https://user-images.githubusercontent.com/5778156/141554052-0bb81827-abdd-41f9-92f3-3a38ef9f08a4.png)